### PR TITLE
[travis] replace tab with 4 spaces in bash script

### PR DESCRIPTION
**Pull request by simonlingoogle**

### CODE REVIEW

File: .travis/before_install.sh [hunk 1]
```diff
@@ -30,8 +30,8 @@
 [ -n "$BUILD_TARGET" ] || exit 0
 
 die() {
-	echo " *** ERROR: " $*
-	exit 1
+    echo " *** ERROR: " $*
```
Comment (bukepo) [inline .travis/before_install.sh:33]:
```suggestion
    echo " *** ERROR:  $*"
```

**Comment by jwhui** [comment]
FYI - Travis-specific scripts will be removed once the migration to GitHub Actions is complete (#4824).

File: .travis/before_install.sh (hunk 1 above)
Comment (simonlingoogle) [reply .travis/before_install.sh:33]: I would prefer this PR to only fix indentation issue.

**Comment by jwhui** [comment]
This file no longer exists.

**Pull request closed without merging**